generation/generate_root_versions_txt.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+GENERATION_DIR=$(dirname -- "$0")
+
+echo "# Format:" >> versions.txt
```

Comment:
Why is it appending? Is there a logic to empty the file first?

Reply:
Oh yes, I forgot to remove the file if it exists before creating it. fixed now.